### DISCUSSION TITLE
Generate/export a css file from tailwind classes?

### DISCUSSION THREAD

**Discussion by mkrkelj1** [tailwindlabs/tailwindcss#4066] [Help]
Is there a way to quickly generate and export a flat css file from tailwind classes? I built a little design mock up tool using tailwind classes, and it would be nice if a user could click a button and download a plain css file of the output. Maybe there is a CLI tool I can call from node to generate the css?

**Comment by MartinHughes-BPC** [comment]
I might be misunderstanding you... but Tailwind generates a CSS file when you run your build process, so presumably you could call that process from your tool?

**Reply by mkrkelj1** [reply]
Poor wording on my part -- but yes, what you're describing is what I'm trying to accomplish, but scoped to a specific component, or string of jsx. So for example, a way to pass something like `<button className="text-lg"> hello world</button>` to a command and return css:

```
{
  font-size: 1.125rem;
  line-height: 1.75rem;
}
```
